vertical-pod-autoscaler/pkg/updater/priority/update_priority_calculator.go

```diff
@@ -177,6 +179,21 @@ func parseVpaObservedContainers(pod *apiv1.Pod) (bool, sets.String) {
 	return hasObservedContainers, vpaContainerSet
 }
 
+func getProcessedRecommendationTargets(r *vpa_types.RecommendedPodResources) string {
+	sb := &strings.Builder{}
+	for _, cr := range r.ContainerRecommendations {
+		sb.WriteString(fmt.Sprintf("%s:", cr.ContainerName))
```

Comment:
if the processed recommendation does not set the CPU or mem for a target, the log for that resource is `0`: `container1: target: 0K 0m; \n` 🤔 this might be a misleading log. @voelzmo WDYT?

Reply:
I tried something else in the latest commit - lmk know what you think!

Reply:
I think you found a good solution to exclude the zero values. Not sure if having it at zero in the logs or having sometimes nothing in the logs for a resource is more confusing for people – I think either way would be ok. Let's see if we get complaints about this ;)

Reply:
I'm also unsure about which one is less confusing! Also wondering - what if the processed recommendation _is actually zero_? Is that possible? Seems like there is no perfect solution 🤔